test/spec/modules/33acrossBidAdapter_spec.js

```diff
@@ -1184,6 +1210,42 @@ describe('33acrossBidAdapter:', function () {
       });
     });
 
+    context('when coppa returns true', function() {
+      it('returns corresponding server requests with coppa: 1', function() {
+        sandbox.stub(config, 'getConfig').withArgs('coppa').returns(true);
+
+        const ttxRequest = new TtxRequestBuilder()
+          .withBanner()
+          .withProduct()
+          .withCoppa(1)
+          .build();
+        const serverRequest = new ServerRequestBuilder()
+          .withData(ttxRequest)
+          .build();
+        const [ builtServerRequest ] = spec.buildRequests(bidRequests, bidderRequest);
+
+        validateBuiltServerRequest(builtServerRequest, serverRequest);
+      });
+    });
+
+    context('when coppa returns false', function() {
+      it('returns corresponding server requests with coppa: 0', function() {
+        sandbox.stub(config, 'getConfig').withArgs('coppa').returns(false);
```

Comment:
most likely getConfig('coppa') will return undefined, that will be the common scenario, so I think you could adjust this spec to cover this case as well.. my recommendation is the following.

e.g.
```
context('when coppa is NOT enabled', function() {
      it('returns corresponding server requests with coppa: 0', function() {
        [false, undefined].forEach((value)=> {
          sandbox.stub(config, 'getConfig').withArgs('coppa').returns(value);
           // the rest of the spec
        })
        
```

Although this "undefined" case scenario might not be needed if we only add coppa to the request when the value is different than undefined, like I am suggesting in the other comment.